codecs/opus/opus-java/pom.xml

```diff
@@ -63,7 +63,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <argLine>-Djava.library.path=../opus-native/${libopus.distro}/target</argLine>
+                    <argLine>-Djava.library.path=../opus-native/${libopus.distro}/target -Drestcomm.opus.library=opus_jni_linux</argLine>
```

Comment:
can you replace `opus_jni_linux` with a variable that is loaded by profile? so we can support macOS as well.